Improve docs for publishing a new version (#452)
alphagov/accessible-autocomplete: CONTRIBUTING.md

- Change included in this release
- Another change included in this release"
```

Then run:
```bash
git push --tags --set-upstream origin refs/heads/v1.2.3:refs/heads/v1.2.3
```

> Open a PR for the release version.

Reviewer: Maybe 'Create a pull request for the release'?

To actually publish, you will need access to an `npm` account that owns `accessible-autocomplete`. Merge the version PR and then run:

```bash
git checkout master
git pull --rebase
npm publish
```

> Create a release in the [Github interface](https://github.com/alphagov/accessible-autocomplete/releases/new)

Reviewer: ```suggestion
[Create a new release](https://github.com/alphagov/accessible-autocomplete/releases/new) on GitHub.
```

Just a tweak to make the link task-based.

>   - select the latest tag version

Reviewer: These are steps so they should be an ordered list rather than an unordered list, and (for GOV.UK style), start with a capital and end with a full stop. For example:

1. Select the latest tag version.
2. Set "v[version-number]" as the title.
etc.

>   - set "v[version-number]" as the title

Reviewer: Not a deal-breaker, but in Frontend docs we've used this style for placeholders:

v\<VERSION-NUMBER\>

>   - add release notes from changelog

Reviewer: ```suggestion
  - add the release notes from the changelog
```

  - add a summary of highlights

>   - (you don't need to manually attach source code files)

Reviewer: As this is a kinda 'non-step', I'd probably add it as a separate sentence after the list. (And remove the brackets).

>   - publish release

Reviewer: ```suggestion
  - publish the release
```

> Send a message to our users in both the X-GOV and GDS #govuk-design-system slack channels that indicates there's a new release with a short summary.

Reviewer: ```suggestion
Post a short summary of the release in the cross-government and GDS #govuk-design-system Slack channels.
```

You might want to borrow this wording that I've used in https://github.com/alphagov/govuk-frontend/pull/1850.

- Change included in this release
- Another change included in this release"
```

Then run:
```bash
git push --tags --set-upstream origin refs/heads/v1.2.3:refs/heads/v1.2.3
```

Create a pull request for the release.

To actually publish, you will need access to an `npm` account that owns `accessible-autocomplete`. Merge the version PR and then run:

```bash
git checkout master
git pull --rebase
npm publish
```

[Create a new release](https://github.com/alphagov/accessible-autocomplete/releases/new) on GitHub.
  1. Select the latest tag version.
  2. Set "v[VERSION-NUMBER]" as the title.
  3. Add the release notes from the changelog.
  4. Add a summary of highlights.
  5. Publish the release.

You don't need to manually attach source code files to the release on GitHub.

Post a short summary of the release in the cross-government and GDS #govuk-design-system Slack channels. For example:

> 🚀 We’ve just released Accessible Autocomplete v2.0.1. You can now [use the acccessible autocomplete multiple times on one page]. Thanks to @<SLACK-NAME> and @<SLACK-NAME> for helping with this release. [https://github.com/alphagov/accessible-autocomplete/releases/tag/v2.0.1](https://github.com/alphagov/accessible-autocomplete/releases/tag/v2.0.1)

Reviewer: Should 'use the accessible autocomplete...' be an actual link? Wasn't sure if it was just an example.

Author: Good catch @m-green, I've removed the square brackets.
